src/migration/9_fix_userdata_columns.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE [dbo].[USERDATA] DROP CONSTRAINT [DF_USERDATA_strSkill]
+
+-- Implicit conversion from data type varchar to binary is not allowed.
+ALTER TABLE [dbo].[USERDATA] ADD [bSkill] BINARY (10) NULL;
+GO
+UPDATE [dbo].[USERDATA] SET [bSkill] = CONVERT(BINARY, [strSkill]);
+GO
+ALTER TABLE [dbo].[USERDATA] DROP COLUMN [strSkill];
+GO
+EXEC sp_rename '[dbo].[USERDATA].[strItem]',  'bItem', 'COLUMN';
+GO
+EXEC sp_rename '[dbo].[USERDATA].[strSerial]',  'bSerial', 'COLUMN';
+GO
```

Comment:
I'll also rework this, since it was my bad recommending on our Discord chat to rename the columns to `bVar` for bytes, whilst my intention was `byVar`.

Reply:
Also done here: https://github.com/ko4life-net/ko-db/pull/12/commits/d5563e3b4cd371986eaa836dff6022ac03d016e7